### PR TITLE
Optimize capacity dashboard

### PR DESCRIPTION
**Pull request by GeorgeH-MS**
1. Add new parameter to generate report only including capacity tiles
2. Add new parameter to filter volume by volume label
3. Add new line in capacity chart to show remaining capacity

### CODE REVIEW

File: Create-AzSStorageDashboard.ps1 [hunk 1]
```diff
@@ -204,6 +224,23 @@ function Get-volumesByType {
 
     $volumesByType = @{}
 
+
```
Comment (GTMer) [inline Create-AzSStorageDashboard.ps1]: Seems no need to write the snippet twice. Could move it after `if ($capacityOnly -eq $True) {`.

File: Create-AzSStorageDashboard.ps1 [hunk 2]
```diff
@@ -204,6 +224,23 @@ function Get-volumesByType {
...
+        infrastructure {
+            $volumeTypes = @("Infrastructure")
+        }
+        vmtemp {
+            $volumeTypes = @("VmTemp")
+        }
+        Default {
+            $volumeTypes = @("ObjStore", "Infrastructure", "VmTemp")
+        }
+
```
Comment (GTMer) [inline Create-AzSStorageDashboard.ps1]: Remove the blank line.

File: Create-AzSStorageDashboard.ps1 [hunk 3]
```diff
@@ -445,7 +501,11 @@ $tileColSpan = 6
 $tileRowSpan = 4
 
 # If you want to add new metrics, adapt function Initialize-TilePsCustomObject and Get-TilePsCustomObject, then register here 
-$metricTypes = @('Throughput', 'Count', 'Latency', 'Capacity')
+if ($capacityOnly -eq $True) {
+    $metricTypes = @('Capacity')
+} else {
+    $metricTypes = @('Capacity', 'Throughput', 'Count', 'Latency')
+}
```
Comment (GTMer) [inline Create-AzSStorageDashboard.ps1:476]: Could add volumeType switch here.

**Comment by GeorgeH-MS** [comment]
recommit code change. Remove volume type definition from Get-volumesByType, and predefine it in Format-Json

**Pull request merged**